docs/docsite/rst/user_guide/become.rst

```diff
@@ -130,28 +133,57 @@ that only allow reading by the user and root, or only allow reading by the unpri
 user being switched to.
 
 However, when both the connection user and the ``become_user`` are unprivileged,
-the module file is written as the user that Ansible connects as, but the file needs to
-be readable by the user Ansible is set to ``become``. In this case, Ansible makes
-the module file world-readable for the duration of the Ansible module execution.
-Once the module is done executing, Ansible deletes the temporary file.
+the module file is written as the user that Ansible connects as (the
+``remote_user``), but the file needs to be readable by the user Ansible is set
+to ``become``. Ansible solves this problem in the following way:
+
+First, if :command:`setfacl` is installed and available in the remote ``PATH``,
+and the temporary directory on the remote host is mounted with POSIX.1e
+filesystem ACL support, Ansible will use POSIX ACLs to share the module file
+with the second unprivileged user.
+
+Next, if POSIX ACLs are **not** available or :command:`setfacl` could not be
+run, Ansible will attempt to change ownership of the module file using
+:command:`chown` for systems which support doing so as an unprivileged user.
+
+If the :command:`chown` fails, Ansible will then check the value of the
+configuration setting ``ansible_common_remote_group``. Many systems will allow a
```

Comment:
'new in 2.10', i know you mention it below but most users won't read that far, so it is good to show version the first time a thing is introduced in the docs